composer.json

```diff
@@ -10,6 +10,7 @@
 	},
 	"require": {
 		"php": "^7.1",
+		"phpstan/phpdoc-parser": "^0.2",
```

Comment:
Can't use `^0.3` until https://github.com/slevomat/coding-standard/pull/534 is complete.

Reply:
Haven't checked what changed... hopefully isn't much...